.evg.yml

```diff
@@ -145,6 +145,7 @@ functions:
   "export variables":
     - command: shell.exec
       params:
+        silent: true
```

Comment:
This is great, but the lines below where we try to print the env are useless now. Maybe we can remove them or comment them saying that since the task is silent it won't be visible?
I am a little unsure why we had to dump it, but it must have been for a good reason.

Reply:
@nbagnard I've removed the env dump lines.
@pmeredit It looks like you added the env dump lines. Were they needed? If so, I will break this up into two functions.

Reply:
They were used in debugging. It's fine to not dump them now.